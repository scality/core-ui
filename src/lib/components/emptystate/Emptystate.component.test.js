import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/emptystate/__snapshots__"
});
