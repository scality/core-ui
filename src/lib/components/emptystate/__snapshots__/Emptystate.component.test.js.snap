// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Emptystate With Link 1`] = `
<div
  className="sc-VigVT gbOMtC"
>
  <div
    className="sc-htoDjs cKPBTk sc-emptystate"
  >
    <div
      className="sc-iwsKbI iKbbyA"
    >
      <div
        className="sc-gqjmRU kuYmln"
      >
        <i
          className="fas fa-server"
        />
      </div>
    </div>
    <div
      className="sc-iwsKbI iKbbyA"
    >
      <h3
        className="sc-dnqmqq fNfZUU"
      >
        A list of 
        nodes
         will appear here
      </h3>
    </div>
    <div
      className="sc-iwsKbI iKbbyA"
    >
      <h3
        className="sc-dnqmqq fNfZUU"
      >
        There are no 
        nodes
         created yet, let's create your first
         
        node
        .
      </h3>
    </div>
    <div
      className="sc-gZMcBi hhVYpv"
    >
      <button
        className="sc-bxivhb dCQlZC sc-button"
        disabled={false}
        onClick={[Function]}
        size="large"
        title=""
        type="button"
      >
        <span
          className="sc-bZQynM iMWVsx"
        >
          <span
            className="sc-button-text"
          >
            <span
              className="sc-ifAKCX joVDVz"
              size="large"
            >
              <i
                className="fas fa-plus"
              />
            </span>
            <span
              className="sc-EHOje boinrg"
            >
              Create node
            </span>
          </span>
        </span>
      </button>
    </div>
  </div>
</div>
`;

exports[`Storyshots Emptystate Without Link 1`] = `
<div
  className="sc-VigVT gbOMtC"
>
  <div
    className="sc-htoDjs cKPBTk sc-emptystate"
  >
    <div
      className="sc-iwsKbI iKbbyA"
    >
      <div
        className="sc-gqjmRU kuYmln"
      >
        <i
          className="fas fa-server"
        />
      </div>
    </div>
    <div
      className="sc-iwsKbI iKbbyA"
    >
      <h3
        className="sc-dnqmqq fNfZUU"
      >
        A list of 
        nodes
         will appear here
      </h3>
    </div>
    <div
      className="sc-iwsKbI iKbbyA"
    >
      <h3
        className="sc-dnqmqq fNfZUU"
      >
        There are no 
        nodes
         created yet, let's create your first
         
        node
        .
      </h3>
    </div>
  </div>
</div>
`;
